src/Identity/UI/src/UIFramework.cs

```diff
@@ -1,21 +1,13 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 namespace Microsoft.AspNetCore.Identity.UI
 {
-    /// <summary>
-    /// The list of supported presentation frameworks for the default UI
-    /// </summary>
-    public enum UIFramework
+    internal enum UIFramework
```

Comment:
Can we avoid making a breaking change here?

Reply:
We want to take all these types out now that we can.